spec/plainyearmonth.html

```diff
@@ -627,26 +627,17 @@ <h1>
         1. Set _other_ to ? ToTemporalYearMonth(_other_).
         1. Let _calendar_ be _yearMonth_.[[Calendar]].
         1. If ? CalendarEquals(_calendar_, _other_.[[Calendar]]) is *false*, throw a *RangeError* exception.
-        1. Set _options_ to ? GetOptionsObject(_options_).
-        1. Let _disallowedUnits_ be « *"week"*, *"day"*, *"hour"*, *"minute"*, *"second"*, *"millisecond"*, *"microsecond"*, *"nanosecond"* ».
-        1. Let _smallestUnit_ be ? ToSmallestTemporalUnit(_options_, _disallowedUnits_, *"month"*).
-        1. Let _largestUnit_ be ? ToLargestTemporalUnit(_options_, _disallowedUnits_, *"auto"*, *"year"*).
```

Comment:
Review if LargerOfTwoTemporalUnits call is missing

Reply:
In the SQ, DifferenceTemporalPlainYearMonth does not call LargerOfTwoTemporalUnits but pass *"year"* to the last arg of  ToLargestTemporalUnit. In my change, I pass *"year"* as the last argument, _smallestLargestDefaultUnit_,  to GetDifferenceSettings . LargerOfTwoTemporalUnits( *"year"* , any unit) is always *"year"* so it is equivlant.